test/dataframerow.jl

```diff
@@ -422,21 +422,21 @@ end
 
     @test sprint(show, DataFrameRow(df, 1, :)) == """
         DataFrameRow
-        │ Row │ a       │ b     │
-        │     │ Nothing │ $(Int) │
-        ├─────┼─────────┼───────┤
-        │ 1   │         │ 1     │"""
+         Row │ a        b
+             │ Nothing  $(Int)
+        ─────┼────────────────
+         1   │          1"""
```

Comment:
I use the same system to print all the columns. Hence the space is necessary. This can be done, but it will require some hack. Is it really necessary? IMHO the output with the space looks nicer.

Reply:
OK, as you like.

Though while we're at it, if `row_number_alignment = :r` allows aligning all row numbers on the right, I'd rather do that instead of `:l` above. It makes much more sense for numbers, and the fact that they are currently left-aligned sounds more like a bug to me.

Reply:
Nice! In fact, I really prefer those numbers to be left aligned (this is the default in PrettyTables). I changed to right because the current printing system of DataFrames.